tachyon/c/math/finite_fields/baby_bear/BUILD.bazel

```diff
@@ -7,14 +7,11 @@ filegroup(
     srcs = ["baby_bear.h"],
 )
 
-tachyon_cc_library(
+generate_small_prime_fields(
```

Comment:
Random question, but don't these bazel targets only general one field at a time? So technically it should be `generate_small_prime_field`?

Reply:
From the perspective of the field, as you said, it's one. But from the perspective of the build targets(files), it's many e.g, The build targets that produce `baby_bear.h`, `baby_bear.cc`, and `baby_bear_types_traits.h` respectively.